bucket/parsecd.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2019.6.19",
+    "description": "Simple, Low-Latency Game Streaming.",
+    "homepage": "https://parsecgaming.com",
+    "license": "Freeware|Proprietary",
+    "notes": "You might need to install ViGEmBus driver (https://github.com/ViGEm/ViGEmBus) for  gamepad support",
+    "architecture": {
+        "64bit": {
+            "url": "https://builds.parsecgaming.com/package/parsec-flat-windows.zip",
+            "hash": "c5e69da443eb0c834e2960928999063326263a638e260076c15346316f40e0ba"
+        },
+        "32bit": {
+            "url": "https://builds.parsecgaming.com/package/parsec-flat-windows32.zip",
+            "hash": "5b7aec1183ce5993419677625854a1cd67fd0cc8aef30ed7a3a88638315baeb1"
+        }
+    },
+    "bin": "parsecd.exe",
+    "persist": [
```

Comment:
- Any of this file do not exists inside archive. Precreate them inside pre_install block
	- Post URL to documentation about each of them
- Move this property after shortcuts